src/config.rs

```diff
@@ -713,6 +727,11 @@ impl Config {
     }
   }
 
+  fn format(ast: Module, search: &Search) -> Result<(), i32> {
+    fs::write(&search.justfile, ast.to_string()).unwrap();
```

Comment:
Trailing newline is added in `Display` for all items. Testing is difficult because in integration tests `\nset dotenv-load := true\n` is manually appended and then stripped, and when `set`s are tested extra newline should be stripped then because of the grouping so I end up with just `trim_end()` for both have and want.

Still this has to be tested. I'll come up with something.